frontend/src/libs/components/header/header.tsx

```diff
@@ -29,7 +29,9 @@ const Header: React.FC<Properties> = ({ user, pageName = '' }: Properties) => {
     <header className={styles['header']}>
       <div className={styles['logo-wrapper']}>
         <Image alt="logo" src={logo} />
-        <span className={styles['logo-text']}>SimpleLand</span>
+        <Link to={AppRoute.ROOT}>
+          <span className={styles['logo-text']}>SimpleLand</span>
```

Comment:
Can't we just attach logo-text `className` to `Link` and get rid of span?

Reply:
maybe because of styles in link component: `color: inherit`
```

.link,
.link:active {
  color: inherit; 
  text-decoration: none;
  border-bottom: 2px solid var(--yellow);
}
```